docs/components/em.md

```diff
@@ -0,0 +1,40 @@
+# matestack core component: Small
```

Comment:
This isn't `small` ;)

Reply:
Haha, the one thing my search & replace missed because it wasn't case sensitive. 😉

Reply:
That's what I would've guessed!